docs/overview.rst

```diff
@@ -0,0 +1,353 @@
+more details on that page, see below.
+
+
+.. _overview-cluster-overview:
+
+Cluster Overview
+----------------
+
+The Cluster Overview page gives a broad range of relevant data for the cluster
+selected.
+
+.. image:: _assets/img/cluster-overview.png
+
+It gives you the following information:
+
+* **Cluster name**: The cluster name.
+
+.. NOTE::
+
+    The cluster name, once set, cannot be changed. It makes up part of the
+    unique cluster URL and is therefore permanent until the cluster is deleted.
+
+* **Cluster ID**: The unique ID of the cluster.
+
+* **Cluster URL**: The URL through which the cluster can be reached. The info
+  icon gives relevant information on ports and access methods. By clicking on
+  the URL you will be redirected to the `CrateDB Admin UI`_ for the cluster at
+  that URL. For more information on the protocols used to connect to the
+  respective ports, refer to the CrateDB Documentation on `HTTP`_ and the
+  `PostgreSQL wire protocol`_ as well as the documentation of your client.
+
+.. NOTE::
+
+    The Cluster URL points to a load balancer that distributes traffic
+    internally to the whole CrateDB cluster. The load balancer closes idle
+    connections after four minutes, therefore client applications that require
+    stateful connections (e.g., JDBC) must be configured to send keep-alive
+    heartbeat queries.
+
+* **Cluster channel**: This indicates what build of CrateDB the cluster is
+  running. The options are stable, testing, or nightly.
+
+* **Version**: This indicates the version number of CrateDB the cluster is
+  running.
+
+* **Created**: The timestamp of the deployment of the cluster.
+
+* **Tier**: This shows what tier of the pricing plan the cluster is running on.
+  For more information on our pricing plans, see the Documentation on `Azure
+  plans`_.
+
+* **RAM/Heap Size**: The currently allocated memory for the cluster.
+
+* **Number of cores**: The number of (or fractional share of) CPU cores
+  dedicated to the cluster.
+
+* **Number of nodes**: The number of nodes the cluster consists of.
+
+* **Storage**: The HD storage capacity allocated to the cluster, for example
+  for backups.
+
+* **Region**: The region the cluster is deployed in.
+
+* **DB Username**: Your username used for login purposes to connect to the
+  cluster. This is set in the configuration during the cluster deployment
+  process.
+
+For more information on some of the terminology used here, refer to the
+`CrateDB architecture Documentation`_.
+
+
+.. _overview-cluster-metrics:
+
+Cluster Metrics
+---------------
+
+The Cluster Metrics page provides live visualizations of how the cluster is
+functioning. It shows two graph panels: one for the average response time of a
+query (in milliseconds) and one for the total number of queries per second.
+
+.. image:: _assets/img/cluster-metrics.png
+
+The differently colored graphs show the contribution of each type of query to
```

Comment:
differently = different

Reply:
But 'colored' is an adjective, not a noun? It's not that the one color is different from the rest, but that they are variously colored. So I think "differently" is right here?

Reply:
Hmm....maybe we should get a third opinion, but "differently" sounds a bit off to me in this context.  Maybe it would help to re-phrase the sentence for clarity?  

i.e. "the contributions of each type of query are shown in different colors in the graphs"

Reply:
Good idea!